packages/api/lib/writeRecords/write-granules.js

```diff
@@ -699,8 +714,8 @@ const writeGranuleFromApi = async (
     published,
     pdrName,
     provider,
-    error = {},
-    createdAt = new Date().valueOf(),
+    error = {}, // Future -- Move this logic to calling methods
+    createdAt,
```

Comment:
This is now being set in the api/message object handling logic at the top of the call chain